pages/profile.php

```diff
@@ -14,14 +14,161 @@
+                Change Picture
+            </a>
+                         </span>
+                     </div>
+                 </div>
+
+                <div class="jumbotron dashboard-title">
+                    <h3>
+            My Profile
+            </h3>
+                </div>
+
+                <h2 class="sub-header">Your Information</h2>
+
+                <div class="table-responsive">
+                    <table class="table table-striped">
+                        <thead>
+                        <tr>
+                            <th>Username</th>
+                            <th>Password</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        <tr>
+                            <td><?php echo $_SESSION["user"]->getUsername(); ?></td>
+                                <td>
+                                    <button type="button" class="btn btn-default" data-toggle="modal" data-target="#password-modal">
+                                        Change Password
+                                    </button>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            <div class="table-responsive">
+                <table class="table table-striped">
+                    <thead>
+                    <tr>
+                        <th>First Name</th>
+                        <th>Last Name</th>
+                        <th>Email Address</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td><?php echo $_SESSION["user"]->getFirstName(); ?></td>
+                        <td><?php echo $_SESSION["user"]->getLastName(); ?></td>
+                        <td><?php echo $_SESSION["user"]->getEmail(); ?></td>
+                    </tr>
+                    </tbody>
+                </table>
+            </div>
+
+            <!-- Password Modal -->
+            <div id="password-modal" class="modal fade" role="dialog">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h4 class="modal-title">Change your password</h4>
+                        </div>
+                        <div class="modal-body">
+                            <form id="changepassword">
+                                <label for="oldpassword" class="sr-only">Old Password</label>
+                                <input name="oldpassword" id="oldpassword" type="password" class="form-control" placeholder="Old Password" autocomplete="off" required autofocus>
+                                <br />
+                                <label for="newpassword" class="sr-only">New Password</label>
```

Comment:
Should be done with css / br just for whitespace violates some (mozilla iirc) rule. On my phone now so can't link